### PR TITLE
Readme.turnserver: how to run server as a daemon

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: README.turnserver [hunk 1]
```diff
@@ -29,6 +29,13 @@ $ ./scripts/secure_relay.sh
 
 RUNNING THE TURN SERVER
 
+To run the coturn server as a daemon use:
+
+  $ turnserver -o
+
+Note that when running coturn as a daemon and you make any changes to the config file the server
+has to be reloaded.
```
Comment (wferi) [inline README.turnserver:37]: If I understood correctly, there's no way to "reload" the daemon (more precisely: the configuration), so it must be **restarted** instead. And it doesn't matter whether coturn is running as a daemon or not.

Comment (misi) [reply README.turnserver:37]: @wferi is right we should say restart not reload config..
Coturn only handles two signals:
* SIGUSR2 reload cert
* HUP reset log file

**Comment by misi** [comment]
You also need to run `./make-man.sh` and `git add man` before `git commit` to update/regenerate the proper man pages too.

**Comment by misi** [comment]
Many Thanks!

**Pull request merged**